src/stories/components/ChecSlideoutPanel.stories.mdx

```diff
@@ -0,0 +1,338 @@
+                  <td>858</td>
+                </tr>
+              </tbody>
+          </ChecTable>
+          <template v-slot:toolbar>
+            <ChecButton
+              variant="small"
+              outline
+              color="red"
+            >
+              Some action
+            </ChecButton>
+            <div class="flex">
+              <ChecButton @click="close" color="primary" text-only class="p-0">
+                Cancel
+              </ChecButton>
+              <ChecButton class="ml-5" color="primary">
+                Save
+              </ChecButton>
+            </div>
+          </template>
+        </ChecSlideoutPanel>
+        <ChecButton @click="open = !open" color="primary" class="m-20">
+          Toggle panel
+        </ChecButton>
+        <h1 class="m-20 font-bold">I am a bunch of content that the panel will slide out on to.</h1>
+        <h2 class="m-20">I am a bunch of content that the panel will slide out on to.</h2>
+        <h3 class="m-20">I am a bunch of content that the panel will slide out on to.</h3>
+        <h4 class="m-20">I am a bunch of content that the panel will slide out on to.</h4>
+        <p class="m-20">I am a bunch of content that the panel will slide out on to.</p>
+        <p class="m-20">I am a bunch of content that the panel will slide out on to.</p>
+      </div>`
+    }}
+  </Story>
+</Preview>
+
+## With stacking panels
+
+<Preview>
+  <Story name="With stacking panels">
+    {{
+      components: {
+        ChecSlideoutPanel,
+        ChecTable,
+        ChecButton,
+      },
+      data() {
+        return {
+          open: false,
+          openSecond: false,
+          openThird: false,
+        };
+      },
+      methods: {
+        close() {
+          this.open = false;
+          return action('Panel closed!')();
+        }
+      },
+      template: `
+        <div>
+          <ChecSlideoutPanel
+            v-if="open"
+            size="half"
+            overlay="light"
+            title="First panel"
+            titleTag="h2"
+            width="2xl"
+            @close="close"
+            closeOnOverlayClick
+          >
+            Hello, I'm the first panel!
+            <ChecSlideoutPanel
+              v-if="openSecond"
+              title="Second panel"
+              size="half"
+              overlay="light"
+              titleTag="h2"
+              width="2xl"
+              @close="openSecond = false"
+            >
+              Hello, I'm the second panel!
+              <ChecSlideoutPanel
+                v-if="openThird"
+                title="Third panel"
+                size="half"
+                overlay="light"
+                titleTag="h2"
+                width="2xl"
+                @close="openThird = false"
+              >
+                Hello, I'm the third panel!
+                <template v-slot:toolbar>
+                  <ChecButton @click="close" color="primary" text-only class="p-0">
+                    Cancel
```

Comment:
I don't like the way this is structured. I think what we want here is siblings rather than nesting:

```
<ChecSlideoutPanel ... />
<ChecSlideoutPanel ... />
<ChecSlideoutPanel ... />
```

z-index will mean they can naturally stack on top of each other, close when you click the overlay or close button, then reveal the next one in the list

Reply:
Will resolve this comment after discussing that nesting makes sense